src/componentes/journal/editor.js

```diff
@@ -26,7 +26,7 @@ export { convertJEditorData2Text };
  
 
 //2020-01-18
-export const JEditor = ({ ymd, range, onClose, saveTrigger, hintTriggerRef, onLoaded, redirect }) => {
+export const JEditor = ({ ymd, range, onClose, saveTrigger, saveTriggerRef, hintTriggerRef, onLoaded, redirect }) => {
```

Comment:
same, saveTrigger and saveTriggerRef hold the same value

Reply:
yes, I wondered about that but couldn't make it work.  I've never used React before.  Sorry for the duplicate PR as well.  I didn't realize this one was created.  It wanted me to sign my commit so I spent some time getting that all set up and committed another one signed.